cpp/oneapi/dal/algo/knn/backend/gpu/infer_kernel_brute_force_dpc.cpp

```diff
@@ -124,8 +153,8 @@ class knn_callback {
     }
 
     sycl::event operator()(std::int64_t qb_id,
-                           const pr::ndview<idx_t, 2>& inp_indices,
-                           const pr::ndview<Float, 2>& inp_distances,
+                           pr::ndview<idx_t, 2>& inp_indices,
+                           pr::ndview<Float, 2>& inp_distances,
```

Comment:
Why do we need to change inputs now?

Reply:
We have to compute `sqrt` for distance voting in case of  `squared_l2_distance`. The easiest way to do so is to make it inplace.